sklearn/utils/setup.py

```diff
@@ -95,6 +95,13 @@ def configuration(parent_package="", top_path=None):
         libraries=libraries,
     )
 
+    config.add_extension(
+        "_heap",
+        sources=["_heap.pyx"],
+        include_dirs=[numpy.get_include()],
```

Comment:
Are you sure the `numpy.get_include()` is necessary ? there's no numpy code in this extension.

Reply:
Nice catch!